core/utils.py

```diff
@@ -35,17 +35,28 @@
 import urllib.request
 import zlib
 
-from core import feconf
 from core.constants import constants
 
 from typing import (
     IO, Any, BinaryIO, Callable, Dict, Iterable, Iterator, List, Optional,
     TextIO, Tuple, TypeVar, Union, overload)
-from typing_extensions import Literal
+
+# Add third_party to path. Some script accesses feconf even before
+# python_libs is added to path.
+_THIRD_PARTY_PATH = os.path.join(os.getcwd(), 'third_party', 'python_libs')
+sys.path.insert(0, _THIRD_PARTY_PATH)
```

Comment:
I think this is the same problem as before, utils.py are also used in the backend code.

Reply:
Resolved.